### PR TITLE
Login: Render success notice to stdout after successful login

### PR DESCRIPTION
**Pull request by grekko**
When retrieving the JWT there is no success message. That may be surprising to users. So let's notify them if all went great.

### CODE REVIEW

**Comment by tisba** [comment]
I cannot find the reference currently, but IIRC a good practise is to minimise output, especially if it is superfluous (since you provide the information anyway).

Because logins are interactive (unless you pass in the password directly), so outputting a bit more would be okay I guess. The reason you want to avoid outputting "success" is that it makes it harder for scripting e.g.

What do you think, @zeisss?

**Comment by grekko** [comment]
> is superfluous (since you provide the information anyway).

you mean b/c the exit code is 0?

> The reason you want to avoid outputting "success" is that it makes it harder for scripting e.g.

right. good point.

**Comment by zeisss** [comment]
Currently:

```
~/p/stormforger/hail $
11:46 $ forge login
No email for login provided
Email: stephan@stormforger.com
Password (will be masked): *************************
```

Well, I can see both points. But the login is mostly done once at the beginning, so output here doesn't cost us much in terms of noise. Also I think I stumbled upon this last year as well.

+1 for adding this.

**Comment by tisba** [comment]
Okay, then go ahead :)

File: cmd/login.go [hunk 1]
```diff
@@ -64,6 +64,9 @@ func runLogin(cmd *cobra.Command, args []string) {
 			err = ioutil.WriteFile(stormforgerConfig, content, 0644)
 			if err != nil {
 				log.Fatal(err)
+			} else {
+				color.White("\nLogin successful!\n\n")
+				color.Red("JWT token stored at %s.\n\n", stormforgerConfig)
 			}
```
Comment (zeisss) [inline cmd/login.go]:
```suggestion
			}
			color.White("\nLogin successful!\n\n")
			color.Red("JWT token stored at %s.\n\n", stormforgerConfig)
```

Comment (zeisss) [reply cmd/login.go]: The `else` is unnecessary, as Fatal() will stop the execution in the error case already.

**Comment by grekko** [comment]
@tisba Ich würde das hier mergen und als `v0.35.1` [`releasen`](https://github.com/stormforger/cli/releases).

**Comment by tisba** [comment]
This is not a bug fix release, so 0.36.0 would be it.

**Comment by grekko** [comment]
> This is not a bug fix release, so 0.36.0 would be it.

ok. out of curiosity. whats the logic for the versioning scheme?

**Pull request merged**